File: SegmentBrazeDebounce-iOS/ObjC/SegmentBrazeDebounce-iOS/ViewController.m
//
//  ViewController.m
//  SegmentBrazeDebounce-iOS
//
//  Created by Brandon Sneed on 11/4/19.
//  Copyright © 2019 Brandon Sneed. All rights reserved.
//

#import "ViewController.h"

@import Analytics;

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextField *textField;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    self.textField.text = @"SomeUser";
}

- (IBAction)sendIdentifyAction:(id)sender {
    [[SEGAnalytics sharedAnalytics] identify:self.textField.text];
}

@end

File: iOS/Swift/SegmentBrazeDebounce-iOS/ViewController.swift
//
//  ViewController.swift
//  SegmentBrazeDebounce-iOS
//
//  Created by Brandon Sneed on 10/22/19.
//  Copyright © 2019 Brandon Sneed. All rights reserved.
//

import UIKit
import Analytics

class ViewController: UIViewController {

    @IBOutlet weak var textField: UITextField!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        textField.text = "SomeUser"
    }

    @IBAction func sendIdentifyAction(_ sender: Any) {
        SEGAnalytics.shared()?.identify(textField.text)
    }
    
}

